### DISCUSSION TITLE
"Oh My Posh Error" Mac terminal

### DISCUSSION THREAD

**Discussion by Longshoez** [JanDeDobbeleer/oh-my-posh#1757] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I followed the instructions on the website but the error  persists

### Theme

jandedobbeleer

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```shell
Oh My Posh Error:                                                                                                  

stat /Users/gabriel/jandedobbeleer.omp.json: no such file or directory
```

**Comment by JanDeDobbeleer** [comment]
@Longshoez seems like that config doesn't exist? Oh-my-posh is reporting that your HOME folder has no file called `jandedobbeleer.omp.json`

**Reply by Longshoez** [reply]
I repeated the isntalation process twice to make sure everything was downloaded, how could this be?

**Reply by JanDeDobbeleer** [reply]
The guide is petty generic in that area and explains that we assume you added that file to that location yourself. On Mac, if you installed via homebrew, you can also use the following command to use an out of the box theme, which is OK, but any adjustments made to that file will be overridden on update thus is discouraged for advanced use-cases:

```bash
eval "$(oh-my-posh --init --shell zsh --config $(brew --prefix oh-my-posh)/themes/jandedobbeleer.omp.json)"
```

**Reply by ChrisFb98** [reply]
> The guide is petty generic in that area and explains that we assume you added that file to that location yourself. On Mac, if you installed via homebrew, you can also use the following command to use an out of the box theme, which is OK, but any adjustments made to that file will be overridden on update thus is discouraged for advanced use-cases:
> 
> ```shell
> eval "$(oh-my-posh --init --shell zsh --config $(brew --prefix oh-my-posh)/themes/jandedobbeleer.omp.json)"
> ```


Thanks a lot, this help me fix de error, im a rookie when it comes about terminal, thanks again!
